utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -434,62 +448,108 @@ int main(int argc, char **argv)
 
                     break;
                 }
-                case 31:
+                case 20:
                 {
-                    testCaseName = "color_cast";
+                    testCaseName = "flip";
 
-                    RpptRGB rgbTensor[batchSize];
-                    Rpp32f alphaTensor[batchSize];
+                    Rpp32u horizontalFlag[batchSize];
+                    Rpp32u verticalFlag[batchSize];
```

Comment:
All comments in HIP test suite should be applicable here as well
Please check and change

Reply:
Done